packages/flutter/lib/src/material/chip.dart

```diff
@@ -1342,13 +1316,14 @@ class _RawChipState extends State<RawChip> with TickerProviderStateMixin<RawChip
     if (!hasDeleteButton) {
       return null;
     }
+    final IconThemeData iconThemeData = theme.iconTheme.copyWith(
+      color: widget.deleteIconColor ?? chipTheme.deleteIconColor,
+    );
     return _wrapWithTooltip(
       new InkResponse(
         onTap: widget.isEnabled ? widget.onDeleted : null,
         child: new IconTheme(
```

Comment:
we should always put child last, makes it clearer what the later arguments (data, etc) relate to.

Reply:
I agree... but isn't that how this is already?

Reply:
oh my i misread this even more than i thought.

I meant for _wrapWithTooltip.

Reply:
Ahh, I see what you mean now.  Fixed.